packages/jsii-reflect/lib/type.ts

```diff
@@ -77,7 +77,7 @@ export abstract class Type implements Documentable, SourceLocatable {
    * Determines whether this is a Data Type (that is, an interface with no methods) or not.
    */
   public isDataType(): this is InterfaceType {
-    return false;
+    return false; // TODO how is this different from isInterfaceType?
```

Comment:
Data types are interfaces with only read-only properties (no methods, no mutable properties).

These are also referred to as *structs*.

Reply:
No, I meant the `isDataType` method versus (`isInterfaceType`). It doesn't seem to do any actual check on if the interface has only readonly properties. When I tried to use this method in the module.ts it didn't do anything.